data-products/_example/01-governance.yml

```diff
@@ -1,23 +1,29 @@
 governance:
-  # owner - who ultimately owns / controls this data (this could be another govenment department)
-  data-platform.justice.gov.uk/owner: "moj"
+  # data-controller - who ultimately owns / controls this data (this could be another govenment department)
```

Comment:
Is data-controller here used as a reference to the legal terms of "data controller" and "data processor"?

Reply:
Yes, that's the intent. Could it be worded better?